app/src/routes/editor/EditPanel/StylesTab/index.tsx

```diff
@@ -104,7 +104,7 @@ const ManualTab = observer(() => {
                 <AccordionTrigger>
                     <h2 className="text-xs font-semibold">{groupKey}</h2>
                 </AccordionTrigger>
-                <AccordionContent>
+                <AccordionContent className="overflow-auto max-h-[50vh]">
```

Comment:
Can you explain why this is necessary? Seems like we just need `overflow-hidden` on the root panel div?

Reply:
It creates issues with the Scroll if I do just that.